_data/library/jsfacades.yml

```diff
@@ -64,6 +64,10 @@
       url: https://github.com/coreyauger/scala-js-youtube
       desc: Type safe and Scala friendly library for the embedded YouTube player API
       dep: ''
+    - name: scala-js-momentjs
+      url: https://github.com/widok/scala-js-momentjs
+      desc: [Moment.js](http://momentjs.com/) facade
```

Comment:
Found it! Add `'` around this line too. Apparently it doesn't like that the value *starts* with a `[`